kernel/userspace.c

```diff
@@ -255,26 +255,26 @@ static void thread_idx_free(uintptr_t tidx)
 
 struct z_object *z_dynamic_object_create(size_t size)
 {
-	struct dyn_obj *dyn_obj;
+	struct dyn_obj *dyn_obj_create;
```

Comment:
this is looking more like a function name than a variable name. I think would be better just `obj` if it is possible.

Reply:
couldn't use obj as it was used for another pointer in the struct. thought this might create confusion.

Reply:
changed it to obj_thread